center/src/main/java/com/microsoft/hydralab/center/interceptor/BaseInterceptor.java

```diff
@@ -64,41 +66,43 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         SecurityContext securityContext = (SecurityContext) request.getSession().getAttribute(HttpSessionSecurityContextRepository.SPRING_SECURITY_CONTEXT_KEY);
         if (securityContext != null) {
             SysUser userAuthentication = (SysUser) securityContext.getAuthentication();
-            oauthToken = userAuthentication.getAccessToken();
+            sessionAuthToken = userAuthentication.getAccessToken();
         }
 
-        String authToken = request.getHeader("Authorization");
-        if (authToken != null) {
-            authToken = authToken.replaceAll("Bearer ", "");
+        authorizationToken = request.getHeader("Authorization");
+        if (authorizationToken != null) {
+            authorizationToken = authorizationToken.replaceAll("Bearer ", "");
         }
 
         // For Azure AD authentication
-        String accessToken = request.getHeader("X-MS-TOKEN-AAD-ID-TOKEN");
+        aadIdToken = request.getHeader("X-MS-TOKEN-AAD-ID-TOKEN");
         LOGGER.info("UserId: " + request.getHeader("X-MS-CLIENT-PRINCIPAL-ID"));
         LOGGER.info("UserName: " + request.getHeader("X-MS-CLIENT-PRINCIPAL-NAME"));
 
-        //check is ignore
+        // check is ignore
         if (!authUtil.isIgnore(requestURI)) {
-            //invoked by API client
-            if (!StringUtils.isEmpty(accessToken)) {
-                if (authTokenService.checkAADToken(accessToken)) {
+            // invoked by API client
+            if (!StringUtils.isEmpty(aadIdToken)) {
+                if (authTokenService.checkAADToken(aadIdToken)) {
                     return true;
                 } else {
                     response.sendError(HttpStatus.UNAUTHORIZED.value(), "unauthorized, error authorization code");
                 }
             }
 
-            //invoke by client
-            if (!StringUtils.isEmpty(authToken)) {
-                if (authTokenService.checkAuthToken(authToken)) {
+            // invoked by agent client
+            if (!StringUtils.isEmpty(authorizationToken)) {
+                if (authTokenService.checkAuthToken(authorizationToken)) {
```

Comment:
Maybe we can add a configuration of center to help decide to use which authType? static authorizationToken or aad authorizationToken